.github/workflows/deploy.yml

```
@@ -83,43 +87,30 @@ jobs:
           profile: minimal
           target: ${{ matrix.target }}
 
-      - name: Setup | musl tools
-        if: contains(matrix.target, 'musl')
-        run: sudo apt install -y musl-tools
-
       - name: Build | Build
-        if: "! contains(matrix.target, 'musl')"
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --release --target ${{ matrix.target }}
+          args: --release ${{ matrix.flags }} --target ${{ matrix.target }}
           use-cross: true
```

Something like this might speed release builds up a bit:
```suggestion
          use-cross: contains(matrix.target, 'musl')
```

```suggestion
          use-cross: contains(matrix.target, 'linux')
```

---

I think given that we will probably have to use cross for the Apple M1 builds and other ARM builds in the future I think it make sense to leave this as it is.

---

Cross doesn't really support cross compiling to MacOS/Windows as far as I can tell.
On my M1 Mac in can cross compile starship to x86_64 without any setup with plain cargo (but cross works too, but it doesn't look like it's doing anything). Windows should also cross-compile without extra setup to ARM64 once sys-info supports it.

But I guess it would make any feature *BSD builds easier.